sr_controllers_tools/src/sr_controllers_tools/sr_controller_helper.py

```diff
@@ -137,3 +148,19 @@ def change_force_ctrl_type(self, chng_type_msg):
             rospy.logwarn("Failed to change the control type.")
 
         return success
+
+    def change_arm_teach_mode(self, teach_mode):
```

Comment:
is this method used anywhere? is teach_mode a boolean?

Reply:
This method is used here: https://github.com/shadow-robot/sr_interface/blob/noetic-devel/sr_robot_launch/scripts/teach_mode_node#L194
Yes, `teach_mode` is a boolean.